### PR TITLE
[zh-cn] Update KubeCon dates

### PR DESCRIPTION
**Pull request by Fan-Lin**
sync #44158

### CODE REVIEW

**Comment by shurup** [comment]
Hi! LGTM! However, I am neither a Chinese language speaker nor a reviewer for this localisation, so let's wait for your team to approve :pray:

File: content/zh-cn/_index.html [hunk 1]
```diff
@@ -71,13 +71,13 @@ <h2>将 150+ 微服务迁移到 Kubernetes 上的挑战</h2>
         <br>
         <br>
         <!-- <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/" button id="desktopKCButton">Attend KubeCon + CloudNativeCon North America on November 6-9, 2023</a> -->
```
Comment (windsonsea) [inline content/zh-cn/_index.html:73]: line 73 shall be updated too

File: content/zh-cn/_index.html [hunk 2]
```diff
@@ -71,13 +71,13 @@ <h2>将 150+ 微服务迁移到 Kubernetes 上的挑战</h2>
...
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/" button id="desktopKCButton">参加 2023 年 11 月 6-9 日的北美 KubeCon + CloudNativeCon</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" button id="desktopKCButton">参加 2024 年 3 月 19-22 日的欧洲 KubeCon + CloudNativeCon</a>
         <br>
         <br>
         <br>
         <br>
         <!-- <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" button id="desktopKCButton">Attend KubeCon + CloudNativeCon Europe on March 19-22, 2024</a> -->
```
Comment (windsonsea) [inline content/zh-cn/_index.html:79]: line 79 shall be updated too

**Comment by windsonsea** [comment]
One question is you need to squash your commits:

https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

**Comment by chenrui333** [comment]
/retitle "[zh-cn] Update KubeCon dates for 2024"

**Comment by chenrui333** [comment]
/retitle [zh-cn] Update KubeCon dates for 2024

**Pull request merged**